### ISSUE TITLE
Add a "loading" icon for pending VMs

### ISSUE THREAD

**Issue by rbf** [slipstream/SlipStreamUI#553]
Related to https://github.com/slipstream/SlipStreamUI/issues/541.

**Comment by rbf** [comment]
@schaubl What exactly is the value `measurement` in the `vms` xml (e.g. at https://nuv.la/vms?media=xml)

```xml
<vms offset="0" limit="20" count="5" totalCount="5" user="bob">
    <vm cloud="cloud-1" user="bob" instanceId="8627" state="Running" measurement="2016-03-14 16:00:32.395 UTC" isUsable="true" cpu="2" ram="1024.0" disk="31.0"/>
    <vm cloud="cloud2" user="bob" instanceId="c55db420-6503-43db-a1ad-4a9473cxxxxx" state="Running" measurement="2016-03-12 10:47:12.850 UTC" runUuid="050808b2-3885-4781-b326-809e6e1exxxx" runOwner="alice" ip="1.1.1.1" name="machine" isUsable="true" cpu="4" ram="8192.0" disk="0.0" instanceType="Large"/>
</vms>
```

To add a `loading` icon for those VMs we have to determine if the VMs has been created less than 15min ago. Can we use this `measurement` with that goal?

**Comment by schaubl** [comment]
@rbf  `measurement` is the time when the VM was seen the first time.
You should use this to determine if the VMs has been created less than 15min ago.

**Comment by rbf** [comment]
Ok, thanks for the clarification.

**Comment by rbf** [comment]
Current implementation:

![screen shot 2016-03-21 at 16 30 50](https://cloud.githubusercontent.com/assets/807084/13923824/5ee4bd1e-ef82-11e5-9a47-996c3f21dc35.png)

- I've added a different icon for _pending_ VMs, i.e. the ones started less than 15min ago with still an unknown run uuid.
- I've also added the icon for non-clickable run uuids to match the ones used in the new detailed info in the gauges, see https://github.com/slipstream/SlipStreamUI/issues/542#issuecomment-197427160.

@schaubl @mebster @cal Any feedback?
